### PR TITLE
Update MiMa to track 10.0.2 and 10.0.3 and update exclusions #870

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #870
* Update MiMa to track 10.0.2 and 10.0.3
* Add exclusion files for akka-http-core, akka-http and akka-http-spray-json

### CODE REVIEW

File: akka-http/src/main/mima-filters/10.0.3.backwards.excludes [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Mima filters needed to check newer versions against 10.0.1
```
Comment (ktoso) [inline akka-http/src/main/mima-filters/10.0.3.backwards.excludes]: against 10.0.2

File: akka-http-marshallers-scala/akka-http-spray-json/src/main/mima-filters/10.0.3.backwards.excludes [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Mima filters needed to check newer versions against 10.0.1
```
Comment (ktoso) [inline akka-http-marshallers-scala/akka-http-spray-json/src/main/mima-filters/10.0.3.backwards.excludes]: against 10.0.2

File: akka-http/src/main/mima-filters/10.0.3.backwards.excludes (hunk 1 above)
Comment (jlprat) [inline akka-http/src/main/mima-filters/10.0.3.backwards.excludes]: copy paste... I'll fix

**Comment by ktoso** [comment]
LGTM, filters look good too

File: project/MiMa.scala [hunk 3]
```diff
@@ -29,7 +29,9 @@ object MiMa extends AutoPlugin {
       // manually maintained list of previous versions to make sure all incompatibilities are found
       // even if so far no files have been been created in this project's mima-filters directory
       Set("10.0.0",
-          "10.0.1")
+          "10.0.1",
+          "10.0.2",
```
Comment (jrudolph) [inline project/MiMa.scala]: Good catch. Forgot to add these in the recent releases.

**Comment by jrudolph** [comment]
Thanks, @jlprat.

**Pull request merged**